src/applications/edu-benefits/0994/pages/benefitsEligibility.js

```diff
@@ -14,8 +14,10 @@ export const uiSchema = {
     'ui:title': '',
     'ui:description': eligibilityInstructions,
     'ui:options': {
-      expandUnder: 'appliedForVaEducationBenefits',
       expandUnderCondition: false,
+      hideIf: formData =>
+        formData.appliedForVaEducationBenefits ||
+        formData.appliedForVaEducationBenefits === undefined,
```

Comment:
A cleaner way to do this would be 
`      hideIf: data => _.get(data, 'appliedForVaEducationBenefits', true),`

Reply:
Thanks for the suggestion, I will go ahead and make that change